issues_to_socrata.py

```diff
@@ -32,12 +33,22 @@ def extract_workgroups_from_labels(labels):
     return ", ".join(workgroup_labels_no_prefix) or None
 
 
-def get_github_issues(repo_name, github_access_token, state="all"):
+def get_github_issues(repo_name, github_access_token, state="open"):
     g = Github(github_access_token)
     repo = g.get_repo(repo_name)
     issues_metadata = repo.get_issues(state=state)
     return [issue for issue in issues_metadata]
 
+def remove_html_comments(text):
+    if not isinstance(text, str):
+        return text  # Return as-is if not a string
+    # Remove HTML comments using regular expression
+    return re.sub(r'<!--(.*?)-->', '', text, flags=re.DOTALL)
+
+def preprocess_issue_description(description):
+    # Apply both transformations
```

Comment:
you mention "both transformations", but i only see one transformation. i think you can remove this function and call `remove_html_comments` directly from the `issue_to_dict` function.

Reply:
definitely! I had another "remove_email_address" function but realized it was not needed, so I deleted it in the end. I'll apply this change